src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days < SHORT_TERM) {
+    return days * dayPrice;
+  }
+
+  if (days >= LONG_TERM) {
+    return days * dayPrice - LONG_TERM_DISCOUNT;
+  }
+
+  return days * dayPrice - SHORT_TERM_DISCOUNT;
```

Comment:
Consider will be better if you create a variable for base price
```suggestion
  const LONG_TERM_DISCOUNT = 50;
  const basePrice = days * dayPrice;

  if (days < SHORT_TERM) {
    return basePrice;
  }

  if (days >= LONG_TERM) {
    return basePrice - LONG_TERM_DISCOUNT;
  }

  return basePrice - SHORT_TERM_DISCOUNT;
```